examples/notebooks/healthcare/beam_post_hl7_messages_to_hcapi.ipynb

```diff
@@ -0,0 +1,504 @@
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following class **BuildMessages** takes the GCS URL from the above class reads it, sepeartes out each message, appends into a list and return list for the next class."
+      ],
+      "metadata": {
+        "id": "Jej68R8w3i2Z"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class BuildMessages(beam.DoFn):\n",
+        "    \"\"\" Class to read file, clean and seperate messgaes based on MSH\"\"\"\n",
+        "    def process(self, file_name):\n",
+        "        try:\n",
+        "            logging.info(\"starting to read file: {}\".format(file_name))\n",
+        "            file = io.gcsio.GcsIO().open(filename=file_name, mode='r')\n",
+        "            read_file = file.read()\n",
+        "            new_file = str(read_file, encoding='utf-8').replace('\\n', '\\r')\n",
+        "            logging.info(\"starting to seperate HL7 messages into list\")\n",
+        "            messages=[]\n",
+        "            for line in new_file.split('\\r'):\n",
+        "                if line[:3] =='MSH':\n",
+        "                    messages.append(line)\n",
+        "                else:\n",
+        "                    messages[-1]+= line\n",
+        "\n",
+        "\n",
+        "            logging.info(\"total number of messages parsed are {}\".format(len(messages)))\n",
+        "            return messages\n",
+        "        except Exception as error:\n",
+        "            logging.error(\"got the following error while processing : {}\".format('\\n'+str(error)))\n",
+        "            raise Exception\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "MC6tr_sGyNKG"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following class **PostToHL7V2Store** takes the messages return in the earlier class and POST each messages to Hl7v2 store ."
+      ],
+      "metadata": {
+        "id": "1hpuoUGA33jo"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class PostToHL7V2Store(beam.DoFn):\n",
+        "    \"\"\" Class to read file, clean and seperate messgaes based on MSH\"\"\"\n",
+        "    def process(self, element):\n",
+        "        try:\n",
+        "            logging.info(\"starting to prepare and post message\")\n",
+        "            hl7v2_store_response = hcapi.message_to_hl7_store(element)\n",
+        "            message_id = hl7v2_store_response['name'].split(\"/\")[-1]\n",
+        "            logging.info(\"successfully posted message to Hl7v2 store with message id :- {}\".format(message_id))\n",
+        "\n",
+        "            # \"Getting the messagge from HL7V2 store for testing purpose\n",
+        "            #message = nb.get_hl7_message(message_id)\n",
+        "            #print(message)\n",
+        "\n",
```

Comment:
Do we need to keep this comment?

Reply:
Yes, incase someone wants to validate and get the message_id from the response, for validation purposes.

Reply:
My preference would probably be to get rid of this (or just always print the whole thing) since a user could do the same thing. If we want to leave it we should at least make the comment clearer - maybe something like: `if you'd like to print the full message instead of just the id, uncomment the following 2 lines`

Also, no matter what note that message is misspelled in the comment

Reply:
@damccorm Removed the code block to avoid any confusion.